driver/flambda_backend_flags.ml

```diff
@@ -354,5 +354,8 @@ let set_o3 () =
 let opt_flag_handler : Clflags.Opt_flag_handler.t =
   { set_oclassic; set_o2; set_o3 }
 
+let use_cached_startup = ref false
+let cached_startup_path = ref (Filename.concat Config.bindir ("cached-startup" ^ Config.ext_obj))
```

Comment:
I don't think we have any other `.o` files in bin directory, `lib/ocaml` may be a better place. Also I would rather have one flag: if the path is hardcoded in the compiler, then the flag can just be boolean. Otherwise, the flag should just have the path, but I don't feel strongly about it.

Reply:
I don't agree with that - if we make the path fully hardcoded, then testing the change might be a bit harder. The other motivation of having two files is to shorten the command line invocation: instead of always passing `-use-xxx /some/path/bin/cached-generic-functions.o` we can now only pass `-use-xxx`